dist/command.js

```diff
@@ -367,6 +368,18 @@ command.hidden = function () {
 };
 
 /**
+ * Allows undeclared options to be passed in with the command.
+ *
+ * @return {Command}
+ * @api public
+ */
+
+command.allowUnknownOptions = function () {
```

Comment:
Can we allow an optional boolean to be passed? It can default to `true`, however if someone wants to modify the rule when important an existing command from an extension, etc. it should be doable.